public/js/test/mochitest/examples/README.md

```diff
@@ -0,0 +1,5 @@
+### Test Examples
+
+##### Pages
+* **doc_script-switching-01** - includes two scripts that reference each other. The second function has a debugger.
+* **doc-scripts** - includes three sources, a long source and two sources that reference each other.
```

Comment:
ha - yeah the `-` and `_` bothered me too. I left it in as i was just removing the code switching example.

I'd like to try keeping the readme up to date if that's okay. With tests it helps me to have a global spec i can see after each run to understand what's being covered. We don't really have that for the examples and it might be nice to skim the list to have a sense. I hate reading one example and then looking at another and trying to keep it all in my head. I can see this getting out of date if we don't remember it, so i'd be happy to abandon the experiment in a week if it doesn't work for us